src/apis/product.apis.ts

```
@@ -63,6 +75,27 @@ export const getOrders = (
 };
+export const getAllOrders = (
+  token: string,
+  page: number = 1,
+  status?: | "PENDING"
+  | "IN_TRANSIT"
+  | "DELIVERED"
+  | "MERCHANT_ACCEPTED"
+  | "MERCHANT_REJECTED"
+) => {
+  let queryParams = "";
+  if (!!status) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)